src/backend/optimizer/plan/planner.c

```diff
@@ -296,7 +295,12 @@ static split_rollup_data *make_new_rollups_for_hash_grouping_set(PlannerInfo *ro
 																 Path *path,
 																 grouping_sets_data *gd);
 
-void compute_jit_flags(PlannedStmt* pstmt);
+static void standard_planner_compute_jit_flags(PlannedStmt* pstmt);
+
+void compute_jit_flags(PlannedStmt* pstmt,
+							double above_cost,
+							double inline_above_cost,
+							double optimize_above_cost);
```

Comment:
How about unifying three cost parameters into a struct?

Reply:
The number of parameters is yet tolerable. So I doubt that it will bring much benefit for now.